RELEASES.md

```diff
@@ -1,3 +1,120 @@
+Version 1.50.0 (2021-02-11)
+============================
+
+Language
+-----------------------
+- [You can now use `const` values for `x` in `[x; N]` array expressions.][79270]
```

Comment:
Not sure how much we want to acknowledge it, but this was already true, just not something we had intentionally stabilized.

Maybe something like "This has been possible since 1.38.0, as it was unintentionally stabilized."